pkg/sqlite/migrationsPostgres/1_initial.up.sql

```diff
@@ -0,0 +1,475 @@
+  gallery_id integer NOT NULL,
+  foreign key(scene_id) references scenes(id) on delete CASCADE,
+  foreign key(gallery_id) references galleries(id) on delete CASCADE,
+  PRIMARY KEY(scene_id, gallery_id)
+);
+CREATE TABLE IF NOT EXISTS galleries_images (
+  gallery_id integer NOT NULL,
+  image_id integer NOT NULL, 
+  cover boolean not null default FALSE,
+  foreign key(gallery_id) references galleries(id) on delete CASCADE,
+  foreign key(image_id) references images(id) on delete CASCADE,
+  PRIMARY KEY(gallery_id, image_id)
+);
+CREATE TABLE IF NOT EXISTS performers_galleries (
+  performer_id integer NOT NULL,
+  gallery_id integer NOT NULL,
+  foreign key(performer_id) references performers(id) on delete CASCADE,
+  foreign key(gallery_id) references galleries(id) on delete CASCADE,
+  PRIMARY KEY(gallery_id, performer_id)
+);
+CREATE TABLE IF NOT EXISTS galleries_tags (
+  gallery_id integer NOT NULL,
+  tag_id integer NOT NULL,
+  foreign key(gallery_id) references galleries(id) on delete CASCADE,
+  foreign key(tag_id) references tags(id) on delete CASCADE,
+  PRIMARY KEY(gallery_id, tag_id)
+);
+CREATE TABLE IF NOT EXISTS performers_tags (
+  performer_id integer NOT NULL,
+  tag_id integer NOT NULL,
+  foreign key(performer_id) references performers(id) on delete CASCADE,
+  foreign key(tag_id) references tags(id) on delete CASCADE,
+  PRIMARY KEY(performer_id, tag_id)
+);
+CREATE TABLE IF NOT EXISTS tag_aliases (
+  tag_id integer NOT NULL,
+  alias varchar(255) NOT NULL,
+  foreign key(tag_id) references tags(id) on delete CASCADE,
+  PRIMARY KEY(tag_id, alias)
+);
+CREATE TABLE IF NOT EXISTS studio_aliases (
+  studio_id integer NOT NULL,
+  alias varchar(255) NOT NULL,
+  foreign key(studio_id) references studios(id) on delete CASCADE,
+  PRIMARY KEY(studio_id, alias)
+);
+CREATE TABLE performer_aliases (
+  performer_id integer NOT NULL,
+  alias varchar(255) NOT NULL,
+  foreign key(performer_id) references performers(id) on delete CASCADE,
+  PRIMARY KEY(performer_id, alias)
+);
+CREATE TABLE galleries_chapters (
+  id serial not null primary key,
+  title varchar(255) not null,
+  image_index integer not null,
+  gallery_id integer not null,
+  created_at timestamp not null,
+  updated_at timestamp not null,
+  foreign key(gallery_id) references galleries(id) on delete CASCADE
+);
+CREATE TABLE scene_urls (
+  scene_id integer NOT NULL,
+  position integer NOT NULL,
+  url varchar(255) NOT NULL,
+  foreign key(scene_id) references scenes(id) on delete CASCADE,
+  PRIMARY KEY(scene_id, position, url)
+);
+CREATE TABLE groups_relations (
+  containing_id integer not null,
+  sub_id integer not null,
+  order_index integer not null,
+  description varchar(255),
+  primary key (containing_id, sub_id),
+  foreign key (containing_id) references groups(id) on delete cascade,
+  foreign key (sub_id) references groups(id) on delete cascade,
+  check (containing_id != sub_id)
+);
+CREATE INDEX index_tags_on_name on tags (name);
```

Comment:
I'd recommend putting the index creation in the same statement as CREATE TABLE (within the brackets), so it's a lot easier to track

Reply:
This is how the original sqlite schema was written, and im not really in favor of diverting too much.

Reply:
Im not gonna change this since it might complicate converting the migrations.